client/test/utils.ts

```diff
@@ -14,3 +16,24 @@ export async function openDoc(docUri: vscode.Uri): Promise<void> {
 async function sleep(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+export function getTestFixtureContent(name: string): Buffer {
+  return readFileSync(path.resolve(__dirname, "../../testFixture", name));
+}
+
+export const assertThrowsAsync = async (fn, expectedMsg?: string) => {
+  try {
+    await fn();
+  } catch (err) {
+    if (expectedMsg) {
+      const typedError: Error = err;
+      assert.include(
+        typedError.message,
+        expectedMsg,
+        "Expected Message not found in returned error message"
+      );
+    }
+    return;
+  }
+  assert.fail("function was expected to throw, but did not");
+};
```

Comment:
Oh. I think I might be able to use this in another pr

Reply:
Cool. Generally I'm not a fan of having files named "utils". It seems like there is a better name we could come up with. To me "utils" doesnt tell a contributor anything about what the functions in there actually _do_. Definitely open to suggestions on better naming!